File: core/src/main/java/br/ufs/nubankchallenge/core/infrastructure/models/ChargebackNoticePayload.java
package br.ufs.nubankchallenge.core.infrastructure.models;

import com.google.gson.annotations.SerializedName;

/**
 * Created by @ubiratanfsoares
 */

public class ChargebackNoticePayload {

    public String title;
    public String description;
    @SerializedName("primary_action") public ActionPayload primaryAction;
    @SerializedName("secondary_action") public ActionPayload secondaryAction;

    public static class ActionPayload {
        public String title;
        public String action;
    }

}

File: core/src/main/java/br/ufs/nubankchallenge/core/infrastructure/models/ChargebackNoticePayload.kt
package br.ufs.nubankchallenge.core.infrastructure.models

import com.google.gson.annotations.SerializedName

/**
 * Created by @ubiratanfsoares
 */

class ChargebackNoticePayload {

    var title: String? = null
    var description: String? = null
    @SerializedName("primary_action") var primaryAction: ActionPayload? = null
    @SerializedName("secondary_action") var secondaryAction: ActionPayload? = null

    class ActionPayload {
        var title: String? = null
        var action: String? = null
    }

}
